Practice/kruglov/home_task_4/PyLec4T5.py

```diff
@@ -0,0 +1,14 @@
+# 5 Interpolation
+
+
+def inter(str1, dict1):
+    if str1[-1:] in dict1:
+        return print("{}{}".format(str1[:-1], dict1[str1[-1:]]))
```

Comment:
It's not correct to return ``print`` since it always returns None.

Reply:
Fixed